scripts/validate

```diff
@@ -35,3 +35,17 @@ done
 
 echo "==> Checking basic shell validation"
 find ../dist -name \*.sh -exec sh -n {} \;
+
+
+echo "==> Add link checker to verify all links in dist are pointing to latest versions"
+
+for LINK_PREFIX in `find ../dist/ -type l -exec basename {} \; | grep sh | sed 's/\.sh//' | sort -V`
```

Comment:
I'm not too versed in bash scripts, but doesn't `find ../dist/ -type l -exec basename {} \;` already return name without extension?
(Though logs from the run seem to disagree with this, could you please explain?)

Reply:
`basename -s` would remove the specified suffix, but these commands are not running a bash shell. They are running in a busybox container and all the commands have very limited functionality. I tried with `basename -s .sh` and it kept throwing errors. Newer busybox containers seem to have fixed it, but the version in use here does not have that fix. So this is more like a work in all scenarios sort of solution.

Reply:
@mitulshah-suse , thank you - I should have googled a bit more. :)